### PR TITLE
added Profile Page

### PR DESCRIPTION
**Pull request by Jasonnyang**
Added profile page component

### CODE REVIEW

File: src/assets/css/ProfilePage.css [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    color: white;
+    align-self: center;
+    width: 135px;
+    height: 55px;
+    border-radius: 30px;
+}
+
+.one-column {
+    height: 450px;
+    width: 390px;
```
Comment (shanekim28) [inline src/assets/css/ProfilePage.css:42]:
```suggestion
    width: 50%;
```

File: src/assets/css/ProfilePage.css [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    align-self: center;
+    border: 1px solid white;  
+    border-radius: 25px;
+    flex-wrap: wrap;
+    
+    
+}
+
+.Save-button {
+    margin-left: 41%;
```
Comment (shanekim28) [inline src/assets/css/ProfilePage.css:25]: This feels pretty dirty; usually we want to keep numbers like these in a way that can be expressed with rational numbers (like 25% = 1/4, 33% = 1/3, etc.). There's probably a much better way to center this.

File: src/assets/css/ProfilePage.css [hunk 3]
```diff
@@ -0,0 +1,64 @@
+.container {
+    display: flex;
+}
+
+.some-page-wrapper {
```
Comment (shanekim28) [inline src/assets/css/ProfilePage.css:5]: Nit: class names should be clear in their function

File: src/assets/css/ProfilePage.css [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+    display: flex;  
+    flex-direction: column;
+    width: 100%;
+    
+}
+
+.column{
```
Comment (shanekim28) [inline src/assets/css/ProfilePage.css:12]: Nit: This class is technically a container, consider changing the name to something that reflects this

File: src/assets/css/ProfilePage.css [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+    padding: 12px 20px;
+    margin: 5px 20px;
+    width: 90%;
+    box-sizing: border-box;
+    border: 1px solid white;
+    background-color: #E0E0E0;
+    border-radius: 5px;
+  }
+
+.second-text {
```
Comment (shanekim28) [inline src/assets/css/ProfilePage.css:61]: Nit: consider naming this class something clearer

File: src/pages/ProfilePage.tsx [hunk 6]
```diff
@@ -0,0 +1,65 @@
+import Sidebar from "../components/Sidebar";
+import "../index.css";
+import React from 'react';
+import ReactDOM from 'react-dom/client';
```
Comment (shanekim28) [inline src/pages/ProfilePage.tsx:4]:
```suggestion
import React from 'react';
import ReactDOM from 'react-dom/client';
```
Remove unused imports

File: src/components/Sidebar.tsx [hunk 7]
```diff
@@ -1,5 +1,6 @@
 import React from "react";
-import "../assets/css/Sidebar.css";
+import "../assets/css/ProfilePage.css";
+import ProfilePage from "../pages/ProfilePage";
```
Comment (shanekim28) [inline src/components/Sidebar.tsx:3]:
```suggestion
import "../assets/css/Sidebar.css"
```
Apply sidebar styling to sidebar and remove unused imports

File: src/assets/css/ProfilePage.css [hunk 8]
```diff
@@ -0,0 +1,64 @@
...
+    width: 100%;
+    
+}
+
+.column{
+    display: flex;
+    height: 607px;
+    width: 784px;  
+    align-self: center;
+    border: 1px solid white;
```
Comment (shanekim28) [inline src/assets/css/ProfilePage.css:17]:
```suggestion
    border: 1px solid white;  
```
Remove border

File: src/assets/css/ProfilePage.css [hunk 9]
```diff
@@ -0,0 +1,64 @@
...
+    align-self: center;
+    width: 135px;
+    height: 55px;
+    border-radius: 30px;
+}
+
+.one-column {
+    height: 450px;
+    width: 390px;
+    border: 1px solid white;
```
Comment (shanekim28) [inline src/assets/css/ProfilePage.css:43]:
```suggestion
    border: 1px solid white;
```
Remove border radius

File: src/pages/ProfilePage.tsx [hunk 10]
```diff
@@ -0,0 +1,65 @@
...
+            <Sidebar />
+            <div className='some-page-wrapper'>
+                <div className="main-content">
+                    <h1 className="content-title">Profile</h1>
+                </div>
+                <div className='column outset'>
+
+                    <div className='one-column inset'>
+                        <h1 className="second-text">Personal</h1>
+                        <form>
```
Comment (shanekim28) [inline src/pages/ProfilePage.tsx:18]: Don't need all these form tags, you can use just one for all the input fields

File: src/pages/ProfilePage.tsx [hunk 11]
```diff
@@ -0,0 +1,65 @@
...
+                            <input type="text" placeholder="First Name" className="text-input" />
```
Comment (shanekim28) [inline src/pages/ProfilePage.tsx:19]:
```suggestion
                            <input type="text" placeholder="First Name" className="text-input inset" />
```
Apply inset styling here and on all other inputs

File: src/pages/ProfilePage.tsx [hunk 12]
```diff
@@ -0,0 +1,65 @@
...
+                        </form>
+
+                        <form>
```
Comment (shanekim28) [inline src/pages/ProfilePage.tsx:22]: Same here, just use one form tag

File: src/pages/ProfilePage.tsx [hunk 13]
```diff
@@ -0,0 +1,65 @@
...
+                        </form>
+                        <form>
+                            <input type="text" placeholder="Ethnicity" className="text-input" />
+                        </form>
+                    </div>
+
+
+                    <div className='one-column inset'>
+                        <h1 className="second-text">Education</h1>
+                        <form>
```
Comment (shanekim28) [inline src/pages/ProfilePage.tsx:42]: And here, just one form tag is good for all of these inputs. Apply inset class to all inputs as well

**Review by shanekim28 — CHANGES_REQUESTED** [review]
Great start! Left some change requests for the bigger stuff I noticed. Anything with "Nit" is optional but highly recommended to look into.

**Comment by Jasonnyang** [comment]
Added new patch and get API functions and fixed the changes above

**Pull request merged**